src/new-components/avatar/avatar.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { render, screen } from '@testing-library/react'
+import React from 'react'
+
+import { Avatar } from './avatar'
+
+describe('Avatar', () => {
+    it('renders a background image when avatarUrl is supplied', () => {
+        render(getAvatar({ avatarUrl: 'https://foo.bar/com.png' }))
+
+        const avatar = screen.getByTestId('avatar')
+
+        expect(avatar).toMatchSnapshot()
+    })
+
+    it('renders initials of user name when avatarUrl is not supplied', () => {
+        render(getAvatar())
+
+        const avatar = screen.getByTestId('avatar')
+
+        expect(avatar).toMatchSnapshot()
```

Comment:
What's the value of these calls to `toMatchSnapshot`? You are already validating via another assertion below what the test case description says (renders initials).

Reply:
Not a clue, I didn't add them, they were already present, so I just left them there.